lib/core/src/main/java/io/atlasmap/core/AtlasMappingService.java

```diff
@@ -38,7 +35,7 @@
 public class AtlasMappingService implements Serializable {
 
     private static final long serialVersionUID = 1668362984516180517L;
-    private static final Logger LOG = LoggerFactory.getLogger(AtlasMappingService.class);
```

Comment:
Again these types of cleanup just make your change set larger.  Perhaps gather these in a subsequent patch.